src/views/troubleshooting.ejs

```diff
@@ -30,7 +30,7 @@
 
   <ul>
     <li class="question">Question: I need to reset my password!</li>
-    <li class="answer">Contact me directly through discord.</li>
+    <li class="answer">You can reset your password <a href="/resetPassword">here</a>.</li>
     <li class="question">Question: I need to change my password!</li>
```

Comment:
Let’s delete this file if we’ve removed the route. Don’t think we need it anymore.